ui/lib/core/addon/components/form-field.js

```diff
@@ -63,8 +63,10 @@ export default class FormFieldComponent extends Component {
     'searchSelect',
     'stringArray',
     'ttl',
+    'toggleButton',
   ];
-  @tracked showInput = false;
+  @tracked showToggleTextInput = false;
+  @tracked toggleInput = false;
```

Comment:
nit: could we rename this to be more explicit?
```suggestion
  @tracked showToggle = false;
```

Reply:
I feel like 'showToggle' doesn't sound like the right description for the basic toggle field

like I changed 'showInput' to 'showToggleTextInput' for the toggle buttons that expect an input value when toggled

And the regular toggle isn't really 'showing' anything, just the input is changing

Reply:
Yeah, that makes sense! I should have been clearer I was more referring to throughout the UI code we typically prepend boolean variables with `has` or `show` to explain a little more what the functionality is i.e. `@tracked showModal = false`. Here it's a little different because it's also literally the "value" of the input, so leaving it as is is fine. Though since FormField is so bloated, a more explicit name like `@tracked toggleInputEnabled = false;` would be clearer.